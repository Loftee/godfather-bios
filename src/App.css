.background {
    width:100%;
    background-color: #2e2e36;
    min-height: 100vh;
}

.header-image {
    width:100%;
    height: 300px;
    background: url('./assets/godfather-header.jpg') center -150px no-repeat;
    background-size: cover;
    position: relative;
}

.header-image__overlay {
    position: absolute;
    width: 100%;
    height: 300px;
    background: rgb(2,0,36);
    background: linear-gradient(0deg, rgba(2,0,36,1) 0%, rgba(46,46,54,0) 30%);
}