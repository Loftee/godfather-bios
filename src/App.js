import React from 'react';
import './App.css';
import Menu from './components/Menu';

function App() {
  return (
    <div className="app">
      <div className={'background'}>
        <div className={'header-image'}>
            <div className={'header-image__overlay'} />
            <div className={'content-container'}>
                <Menu />
            </div>
        </div>
      </div>
    </div>
  );
}

export default App;
