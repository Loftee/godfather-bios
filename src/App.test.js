import React from 'react';
import {shallow} from 'enzyme';
import App from './App';

describe('App', () => {
    const app = shallow(<App />);

    it('renders correctly', () => {
        expect(app).toMatchSnapshot();
    });

    it('has a menu component', () => {
        expect(app.find('Menu').exists()).toBe(true);
    });

    it('has a SelectedEmployee component', () => {
        expect(app.find('SelectedEmployee').exists()).toBe(true);
    });
});
