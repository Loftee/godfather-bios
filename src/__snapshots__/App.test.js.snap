// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div
  className="app"
>
  <div
    className="background"
  >
    <div
      className="header-image"
    >
      <div
        className="header-image__overlay"
      />
      <div
        className="content-container"
      >
        <Menu />
      </div>
    </div>
  </div>
</div>
`;
